### ISSUE TITLE
The clear tags 'x' button on the blogs page needs an alt text

### ISSUE THREAD

**Issue by steven1046** [OpenLiberty/openliberty.io#1310]
The validator tool found one violation on the blogs page where the x button to clear tags needs alt text: https://validator.w3.org/nu/?doc=https%3A%2F%2Fopenlibertydev.mybluemix.net%2Fblog%2F
